internal/policies/mount/mount_test.go

```diff
@@ -73,24 +95,48 @@ func TestApplyPolicy(t *testing.T) {
 
 		// Special cases.
-		"user, successfully apply policy with anonymous values":     {entries: []string{"entry with anonymous tags"}},
-		"user, creates only users_user dir if the entry is errored": {entries: []string{"errored entry"}},
+		"user, successfully apply policy with anonymous values": {entries: []string{"entry with anonymous tags"}},
+		"user, creates only dirs if the entry is errored":       {entries: []string{"errored entry"}},
 
 		// Badly formatted entries.
 		"user, successfully apply policy trimming whitespaces":           {entries: []string{"entry with spaces"}},
 		"user, successfully apply policy trimming sequential linebreaks": {entries: []string{"entry with multiple linebreaks"}},
-		"user, creates only users_user dir if the entry is empty":        {entries: []string{"entry with no value"}},
-		"user, creates only users dir if there are no entries":           {entries: []string{"no entries"}},
+		"user, creates only dirs if the entry is empty":                  {entries: []string{"entry with no value"}},
+		"user, creates only dirs if there are no entries":                {entries: []string{"no entries"}},
 
 		// Policy refresh.
 		"user, mount file is removed on refreshing policy with no entries":                    {secondCall: []string{"no entries"}},
 		"user, mount file is removed on refreshing policy with an empty entry":                {secondCall: []string{"entry with no value"}},
 		"user, mount file is updated on refreshing policy with an entry with multiple values": {secondCall: []string{"entry with multiple values"}},
 
+		/**************************** SYSTEM ***************************/
+		// Success cases.
+		"system, successfully apply policy for entry with one value":        {isComputer: true},
+		"system, successfully apply policy for entry with multiple values":  {entries: []string{"entry with multiple values"}, isComputer: true},
+		"system, successfully apply policy for entry with repeatead values": {entries: []string{"entry with repeatead values"}, isComputer: true},
+		"system, successfully apply policy filtering out unsupported keys":  {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "system-mounts"}, isComputer: true},
+
+		// Special cases.
+		"system, successfully apply policy with anonymous values": {entries: []string{"entry with anonymous tags"}, isComputer: true},
+		"system, does nothing if the entry is errored":            {entries: []string{"errored entry"}, isComputer: true},
+
+		// Badly formatted entries.
+		"system, successfully apply policy trimming whitespaces":           {entries: []string{"entry with spaces"}, isComputer: true},
+		"system, successfully apply policy trimming sequential linebreaks": {entries: []string{"entry with multiple linebreaks"}, isComputer: true},
```

Comment:
The golden files for that one seems empty too. is it intended?

Reply:
I think your github might have bugged or something... The dirs look like this to me:
- [system, successfully apply policy trimming whitespaces](https://github.com/ubuntu/adsys/tree/system-mounts-policy-manager/internal/policies/mount/testdata/TestApplyPolicy/system%2C_successfully_apply_policy_trimming_whitespaces/etc/systemd/system)
- [system, successfully apply policy trimming sequential linebreaks](https://github.com/ubuntu/adsys/tree/system-mounts-policy-manager/internal/policies/mount/testdata/TestApplyPolicy/system%2C_successfully_apply_policy_trimming_sequential_linebreaks/etc/systemd/system)

Reply:
ditto, I don’t know what happens with the diff that was presented to me. Sorry for the noise!